_opensearch/rest-api/analyze-apis/perform-text-analysis.md

```diff
@@ -0,0 +1,670 @@
+  "explain" : true,
+  "attributes" : ["keyword"] 
+}
+````
+
+The previous request returns the following fields:
+
+````json
+{
+  "detail" : {
+    "custom_analyzer" : true,
+    "charfilters" : [ ],
+    "tokenizer" : {
+      "name" : "standard",
+      "tokens" : [
+        {
+          "token" : "OpenSearch",
+          "start_offset" : 0,
+          "end_offset" : 10,
+          "type" : "<ALPHANUM>",
+          "position" : 0
+        },
+        {
+          "token" : "analyze",
+          "start_offset" : 11,
+          "end_offset" : 18,
+          "type" : "<ALPHANUM>",
+          "position" : 1
+        },
+        {
+          "token" : "test",
+          "start_offset" : 19,
+          "end_offset" : 23,
+          "type" : "<ALPHANUM>",
+          "position" : 2
+        }
+      ]
+    },
+    "tokenfilters" : [
+      {
+        "name" : "reverse",
+        "tokens" : [
+          {
+            "token" : "hcraeSnepO",
+            "start_offset" : 0,
+            "end_offset" : 10,
+            "type" : "<ALPHANUM>",
+            "position" : 0
+          },
+          {
+            "token" : "ezylana",
+            "start_offset" : 11,
+            "end_offset" : 18,
+            "type" : "<ALPHANUM>",
+            "position" : 1
+          },
+          {
+            "token" : "tset",
+            "start_offset" : 19,
+            "end_offset" : 23,
+            "type" : "<ALPHANUM>",
+            "position" : 2
+          }
+        ]
+      }
+    ]
+  }
+}
+````
+
+#### Set a token limit
+
+You can set a limit to the number of tokens generated. Setting lower values reduces a node's memory usage. The default value is 10000.
+
+The following request limits the tokens to four:
+
+````json
+PUT /books2
+{
+  "settings" : {
+    "index.analyze.max_token_count" : 4
+  }
+}
+````
+The preceding request is an index API rather than an analyze API. See [DYNAMIC INDEX SETTINGS]({{site.url}}{{site.baseurl}}/opensearch/rest-api/index-apis/create-index/#dynamic-index-settings) for additional details.
+{: .note}
+
+### Response fields
+
+The text analysis endpoints return the following response fields:
+
+Field | Data Type | Description
+:--- | :--- | :---
+tokens | Array | Array of tokens derived from the `text`. See [token object](#token-object).
+detail | Object | Details about the analysis and each token. Included only when you request token details. See [detail object](#detail-object).
+
+#### Token object
+
+Field | Data Type | Description
+:--- | :--- | :---
+token  | String | The token's text.
+start_offset | Integer | Token's starting position within the original text string. Offsets are zero-based.
+end_offset | Integer | Token's ending position within the original text string.
+type | String | Classification of the token. `<ALPHANUM>`, `<NUM>`, and so on. The tokenizer usually sets the type, but some filters define their own types. For example, the synonym filter defines the `<SYNONYM>` type.
```

Comment:
Would it make sense for the first period to be a colon, as it's introducing the classifications? I feel like the first two sentences should be connected in some way.